test/controllers/home_controller_test.rb

```diff
@@ -0,0 +1,7 @@
+require "test_helper"
+
+class HomeControllerTest < ActionDispatch::IntegrationTest
+  # test "the truth" do
+  #   assert true
+  # end
```

Comment:
If you dont need thisdelete it 
Don leave white comments
```suggestion

```

Reply:
done